images/scripts/fedora.setup

```diff
@@ -18,6 +18,7 @@ echo foobar | passwd --stdin root
 # the distribution repository is updated.
 #
 COCKPIT_DEPS="\
+acl \
```

Comment:
This should have gone into `TEST_DEPENDS`, but let's do that in a follow-up (if nothing else has to change)

Reply:
Good catch.. working on a follow up.